app/views/collections/form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render Elements::HeaderComponent.new(level: :h1, variant: :h2, text: @collection_form.title || 'Untitled collection', classes: 'mb-3') %>
+<%= render Elements::AlertComponent.new(variant: :danger, value: I18n.t('collections.edit.errors.validation')) if @collection_form.errors.present? %>
+<%= render Elements::Layout::TabbedFormComponent.new do |component| %>
+  <% component.with_tab(label: I18n.t('collections.edit.panes.title.tab_label'), tab_name: :title, selected: true) %>
+  <% component.with_tab(label: I18n.t('collections.edit.panes.description.tab_label'), tab_name: :description) %>
+  <% component.with_tab(label: I18n.t('collections.edit.panes.related_content.tab_label'), tab_name: :related_content) %>
+  <% component.with_tab(label: I18n.t('collections.edit.panes.deposit.tab_label'), tab_name: :deposit) %>
+  <% component.with_form_collection(collection_form: @collection_form) %>
```

Comment:
It seems like the tabs and the panes should be defined in the same place instead of 2 different places.